dnf-plugins-core.spec

```diff
@@ -26,7 +35,8 @@ protected_packages passive plugins.
 %package -n python-dnf-plugins-core
 Summary:    Core Plugins for DNF
 Group:      System Environment/Base
-BuildRequires:  python-dnf >= %{dnf_version}
+BuildRequires:  python-dnf >= %{dnf_lowest_compatible}
+BuildRequires:  python-dnf < %{dnf_not_compatible}
```

Comment:
Sure but imagine this situation (which already happened):
1. a user has both ```dnf-0.6``` and ```dnf-plugins-core-0.1``` installed
2. then we release ```dnf-1.0``` and ```dnf-plugins-core-1.0``` (e.g. together if you want)
3. but the user runs just ```dnf upgrade dnf``` or ```dnf upgrade``` but e.g. ```dnf-plugins-core-1.0``` has some broken dependencies
4. the user now has ```dnf-1.0``` and ```dnf-plugins-core-0.1``` (!) installed
5. because ```dnf-plugins-core-0.1``` uses the old API, it kills DNF now

But with the ```< %{dnf_not_compatible}``` above, the user even would not be able to upgrade only ```dnf```. She would have to upgrade both or none.